components.md

```diff
@@ -292,6 +293,129 @@ _to be written_
+
+### Basic Implementation
+
+```svelte
+<script>
+  let value = localStorage.getItem("name") || "";
+
+  function saveToLocal() {
+    localStorage.setItem("name", value);
+  }
+</script>
+
+<div>
+  <h1>Svelte with Basic Local Storage</h1>
+  <input type="text" bind:value />
+  <button on:click="{saveToLocal}">Save to Local</button>
+  <p>{value}</p>
+</div>
+```
+
+Type something in the `input` field and hit the `button`. When you refresh the page, voila the data has persisted and is used as the default value for the `value` prop.
+
+### More Complex Values
+
+Since Local Storage only works with simple values, more complex values like objects or arrays must be serialized and deserialized with JSON in order to store them in Local Storage.
+
+```svelte
+<script>
+  import { onMount } from "svelte";
+
+  let todos = [];
+  let newTodo;
+
+  function addTodo() {
+    // ensure they actually typed something
+    if (!newTodo) {
+      return;
+    }
+
+    todos = [...todos, newTodo];
+    newTodo = "";
+    saveTodosToLocal();
+  }
+
+  function removeTodo(index) {
+    const length = todos.length;
+    todos = [...todos.slice(0, index), ...todos.slice(index + 1, length)];
+    console.log(todos);
+    saveTodosToLocal();
+  }
+
+  function saveTodosToLocal() {
+    const parsed = JSON.stringify(todos);
+    localStorage.setItem("todos", parsed);
+  }
+
+  onMount(() => {
+    if (localStorage.getItem("todos")) {
+      try {
+        todos = JSON.parse(localStorage.getItem("todos"));
+      } catch (e) {
+        localStorage.removeItem("todos");
+      }
+    }
+  });
+</script>
+
+<div>
+  <h2>Todo List</h2>
+  <ul>
+    {#each todos as todo, i}
+    <li>
+      <span>{todo}</span>
+      <button on:click={() => removeTodo(i)}>Remove Todo</button>
+    </li>
+    {/each}
+  </ul>
+  <p>
+    <input type="text" bind:value={newTodo} />
+    <button on:click={addTodo}>Add Todo</button>
+  </p>
+</div>
+```
+
+Here the data is initialized as an empty array and then `onMount` is used to get any prior data stored in Local Storage. Since the data has to be deserialized first this approach is used instead of the previous version where the data was initialized with any existing Local Storage result.
+
+The rest is pretty standard Svelte Todo app business.
+
+### A Persistent Svelte Store
+
+For cases when you need more than localized component state, Svelte offers _stores_. Svelte stores, however, do not persist data. We're going wrap the svelte `writable` store implementation with some Local Storage functionality.
+
+```js
+// localStorageStore.js
+import { writable, get } from "svelte/store";
+
```

Comment:
I don't see the localStorage assignment

```suggestion
const ls = localStorage
```
or update `ls` to `localStorage`

Reply:
Ooof. Thanks for catching that.